static/js/publisher/release/actions/availableRevisionsSelect.js

```diff
@@ -6,3 +14,34 @@ export function setAvailableRevisionsSelect(value) {
     payload: { value }
   };
 }
+
+export function selectAvailableRevisions(value) {
+  return (dispatch, getState) => {
+    dispatch(setAvailableRevisionsSelect(value));
+    dispatch(clearSelectedRevisions());
+
+    const state = getState();
+
+    // for each architecture
+    const archs = getArchitectures(getState());
+    let revisionsFilter = () => true;
+
+    // for Recent select only revisions from most recent uploaded version
+    if (value === AVAILABLE_REVISIONS_SELECT_RECENT) {
+      // find most recent version
+      const recentVersion = getSelectedAvailableRevisions(state)[0].version;
```

Comment:
A shallow look at `getSelectedAvailableRevisions` it seems `[0]` *could* be `undefined`?

Reply:
Good catch. Shouldn't happen in practice (as you cannot select option with 0 revisions), but it is indeed a possible state. Will fix that. Thx.